static/css/misc.less

```diff
@@ -1030,24 +1030,23 @@
     - cfgov-misc
 */
 
+
+// Size of cicular background.
+@size: 1.5em;
```

Comment:
I like @icon-size because if you change the units you don't have to change the variable name.